DESCRIPTION

```diff
@@ -31,12 +31,15 @@ Depends:
     shinyBS
 Imports:
     config,
+    dplyr,
     ggplot2,
     glue,
     golem,
     htmltools,
     knitr,
+    magrittr,
     markdown,
+    pkgload,
```

Comment:
I don't see where this is used, and R CMD check produces a `NOTE` saying that it's not used. I think we can remove

Reply:
The note was a little annoying so I added it in. 😆 But it's a good point that it's not needed for the package itself so users shouldn't have to have it. I went ahead and removed for now.